code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -600,7 +600,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 			if("Kidneys")
 				organ = BP_KIDNEYS
 
-		var/list/organ_choices = list("Normal","Assisted","Synthetic")
+		var/list/organ_choices = list("Normal")
```

Comment:
seems like assisted ones should be allowed

Reply:
Assisted ones may not properly process chems, is the issue.